src/cloud-cli/meltano/cloud/README.md

```diff
@@ -0,0 +1,85 @@
+# Meltano Cloud CLI
+
+`meltano-cloud` is a standalone CLI tool for interacting with the Meltano Cloud API.
+
+## Configuration
+
+`meltano-cloud` can be configured via
+`meltano-cloud config set <setting name> <setting value>`.
+
```

Comment:
I was thinking that we would preserve `meltano-cloud config set` for project settings and secrets that would be stored in the cloud.

Is that still the case, or would this also be used for the below options like `auth_callback_port`?

We need to think a bit about collisions and behavior of get/set if so.

Reply:
That makes sense -- this line in the docs is actually outdated, as we had settled in another discussion on configuring exclusively via env vars and flags, since there are very few cloud-specific config options. I'll remove this here.